src/main/java/org/jitsi/jicofo/recording/jibri/JibriRecorder.java

```diff
@@ -357,7 +357,7 @@ public void run()
                     }
                     //else the response will be handled in processPacket()
                 }
-                catch (OperationFailedException e)
+                catch (Throwable e)
```

Comment:
OperatonFailedException was caught from sendPacketAndGetReply and the only case for that one there is when XMPP is disconnected. So the only thing logged there is a message that the connection is broken. For Throwable we will want to log the whole stack trace I guess.